components/gitpod-db/src/typeorm/entity/db-gitpod-token.ts

```diff
@@ -24,9 +24,8 @@ export class DBGitpodToken implements GitpodToken {
     @Column({ type: "int" })
     type: GitpodTokenType;
 
-    @ManyToOne((type) => DBUser)
-    @JoinColumn()
-    user: DBUser;
+    @Column(TypeORM.UUID_COLUMN_TYPE)
+    userId: string;
```

Comment:
This is the main change, i.e. we remove the ORM relationship mapping and replace it with a simple foreign key exposure.